source/whats-new.txt

```diff
@@ -58,6 +58,24 @@ What's New in 6.0
      import these methods from the driver instead of from
      ``mongodb-client-encryption``.
 
+   - Removed support for strings with a length of 12 from the ``ObjectId``
+     constructor.
+
+   - The ``abortTransaction`` and ``commitTransaction`` methods return
+     ``null`` instead of the raw command results.
+
+   - Removed connection option helpers that recognized values other than ``true``
+     or  ``false`` as booleans. You must provide either ``true`` or ``false`` values for
+     options that expect boolean values in the connection string or MongoClient
+     constructor.
```

Comment:
Suggestion:

I think this might be a more direct way to explain the new requirement:

```suggestion
     or  ``false`` as booleans. You must provide either ``true`` or ``false`` values in
     the connection string or to the MongoClient constructor.
```